### PR TITLE
Add `pywin32` and apply platform constants

### PR DESCRIPTION
**Pull request by QU3B1M**
|Related Issue|
|---|
| https://github.com/wazuh/qa-integration-framework/issues/15|

## Description

This PR applies the platform constants on all the _platform conditionals_, it also adds a default timeout when the tests are being executed on Windows and gives 10 more seconds to the `FileMonitor` default monitor.

### CODE REVIEW

File: src/wazuh_testing/tools/file_monitor.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def __validate_parameters(self) -> None:
         if not os.access(self.monitored_file, os.R_OK):
             raise PermissionError(f"{self.monitored_file} is not readable.")
 
-    def start(self, callback: Callable, timeout: int = 10, accumulations: int = 1, only_new_events: bool = False) -> None:
+    def start(self, callback: Callable, timeout: int = 30, accumulations: int = 1, only_new_events: bool = False) -> None:
```
Comment (TomasTurina) [inline src/wazuh_testing/tools/file_monitor.py:55]: Why 30, isn't 10 enough?

File: src/wazuh_testing/utils/configuration.py [hunk 2]
```diff
@@ -323,8 +324,8 @@ def load_configuration_template(data_file_path, configuration_parameters=[], con
 
     configuration = file.read_yaml(data_file_path)
 
-    if sys.platform == 'darwin':
-        configuration = set_correct_prefix(configuration, PREFIX)
+    if sys.platform == MACOS:
+        configuration = set_correct_prefix(configuration, PREFIX) # TODO: FIX THIS.
```
Comment (TomasTurina) [inline src/wazuh_testing/utils/configuration.py:328]: What is this comment?

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: src/wazuh_testing/tools/file_monitor.py (hunk 1 above)
Comment (QU3B1M) [reply src/wazuh_testing/tools/file_monitor.py:55]: In the case of windows running on Actions nope, local it works correctly with 10s, but on the workflow it fails

File: src/wazuh_testing/utils/configuration.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 
 from wazuh_testing import DATA_PATH
 from wazuh_testing.constants.paths.configurations import WAZUH_CONF_PATH, WAZUH_LOCAL_INTERNAL_OPTIONS
+from wazuh_testing.constants.platforms import MACOS
```
Comment (TomasTurina) [inline src/wazuh_testing/utils/configuration.py:13]: Not necessary anymore.

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**